src/main/java/com/mastercard/app/petstore/utils/MockDataBuilders.java

```diff
@@ -17,7 +18,7 @@ public static NewCat buildNewCat(){
         newCat.setBreed("Tabby");
         newCat.setColor("White with black spots");
         newCat.setGender("FEMALE");
-        newCat.birthdate(new Birthdate().year(2022).month(04).day(17));
+        newCat.birthdate(new Birthdate().year(2022).month(4).day(17));
```

Comment:
How come this is changed from '04' to '4', is date format not **_YYYY/MM/DD_**?

Reply:
It's the same thing. If this was taking in a string like "2022-04-07" then it would matter but this is an Int getting set so it doesn't.